dockerfiles/init/manifests/che.env

```diff
@@ -122,6 +122,11 @@
 #     agents within the workspace when it boots, then the workspace will not be started.
 #CHE_WORKSPACE_AGENT_DEV_MAX__START__TIME__MS=300000
 
+# Workspace logs root
+#     Defines the directory inside the machine where all the workspace logs are placed.
+#     The value of this folder should be provided into machine e.g. like environment variable
+#     so agents developers can use this directory for backup agents logs.
+#CHE_WORKSPACE_LOGS_ROOT_DIR=/workspace_logs
```

Comment:
maybe default value should be set right here in `che.env` instead of defining in each agent installer script? 
This way it will be more obvious to  end user where are logs stored by default

Reply:
By default value from `che.properties` will be used, and it equals to `/workspace_logs`(we won't define it in each script). This value is commented to be the same as all properties in this file.

Reply:
My point was just to let user know what default is. So if default defined in che.props can you at least mention about that here in `che.env`

Reply:
ok, thx